### PR TITLE
added details on XO orientation for nano rx 

### PR DESCRIPTION
**Pull request by hmalaspina**
Added details on XO orientation for nano RX and changed the cheat sheet accordingly.

The XO orientation changed on PCB v1.2.
Did not realised this and it caused me to pull my hair for 1 week since everything else seemed to work. The ESP was even able to read the revision from the sx1280.

Maybe this PR would avoid this struggle to other people.

### CODE REVIEW

**Comment by JBKingdon** [comment]
Looks good to me. @JyeSmith would be the one to have the final say. You can see the new cheat sheet at https://github.com/hmalaspina/ExpressLRS-Hardware/tree/nano-rx-xo-orientation/PCB/2400MHz/RX_Nano

**Comment by JyeSmith** [comment]
I am not aware that these XOs have a required orientation.

**Comment by JBKingdon** [comment]
Long ways :grin: We're used to it, but not everyone is.

File: PCB/2400MHz/RX_Nano/README.md [hunk 1]
```diff
@@ -61,4 +61,7 @@ https://youtu.be/bOKTOFyyGHo
 
 I made the below image as a quick reference to help when placing components.
 
+Pay attention to the crystal's placement. Orientation differs on older PCBs.
+In any case you want the crystal's pin number 1 to be grounded.
```
Comment (JBKingdon) [inline PCB/2400MHz/RX_Nano/README.md:65]: Oh, I just checked the datasheet, pin 1 is marked as xtal, it's pins 2 and 4 that are ground

Comment (hmalaspina) [reply PCB/2400MHz/RX_Nano/README.md:65]: Good catch will fix.

**Comment by JyeSmith** [comment]
> Long ways 😁 We're used to it, but not everyone is.

Gotcha 😄

**Pull request merged**